Project-Aurora/Project-Aurora/Settings/Overrides/Logic/Boolean/Boolean_FlipFlop.cs

```diff
@@ -0,0 +1,77 @@
+﻿using Aurora.Profiles;
+using Aurora.Utils;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Media;
+
+namespace Aurora.Settings.Overrides.Logic.Boolean {
+
+    /// <summary>
+    /// A simple memory gate that can be used for storing a boolean state.
+    /// While the given input is true, the state of the flip-flop is toggled.
+    /// </summary>
+    [Evaluatable("Flip-flop (Toggle)", category: EvaluatableCategory.Logic)]
+    public class Boolean_FlipFlopT : Evaluatable<bool> {
+
+        private bool state = false;
+
+        public Evaluatable<bool> Toggle { get; set; }
+
+        public Boolean_FlipFlopT() : this(EvaluatableDefaults.Get<bool>()) { }
+        public Boolean_FlipFlopT(Evaluatable<bool> toggle) => Toggle = toggle;
+
+        protected override bool Execute(IGameState gameState) {
+            if (Toggle.Evaluate(gameState))
```

Comment:
It should only toggle if it got false since last toggle. I know you can "fix" it with the boolean change detector but I still think this change should be done. Everyone that worked with graphical programming before will expect it works like that.